inferno/callbacks.py

```diff
@@ -267,6 +292,10 @@ class PrintLog(Callback):
     `BestLoss` callback takes care of creating those entries, which is
     why `PrintLog` works best in conjunction with that callback.
 
+    *Note*: `PrintLog` will not result in good outputs if the number
```

Comment:
Not sure here. In Python docstrings, I prefer to use "yield" specifically to mean that a function `yield`s things, which is not the case here. Would the change result in more clarity?